server/controllers/api/videos/index.ts

```diff
@@ -184,17 +203,38 @@ async function addVideo (req: express.Request, res: express.Response) {
 
   const videoPhysicalFile = req.files['videofile'][0]
   const videoInfo: VideoCreate = req.body
+  const files = req.files
+
+  return addVideo(req, res, { videoPhysicalFile, videoInfo, files })
+}
 
-  const videoData = buildLocalVideoFromReq(videoInfo, res.locals.videoChannel.id)
+async function addVideoResumable (req: express.Request, res: express.Response) {
+  const videoPhysicalFile = res.locals.videoFileResumable
+  const videoInfo = videoPhysicalFile.metadata
+  const files = { image: { path: getResumableUploadPath(videoPhysicalFile.filename) } }
```

Comment:
I don't understand what is `image`, and why we assign the path of the video

Reply:
I'm pretty sure this has to do with the audio background image being set for _some_ requests, but that should be clarified by @kontrollanten (who named it 'audioBg' initially).

Reply:
A lot has changed since I implemented this. As far as I remember `files` was sent to a function that ran `Object.keys(files).forEach(...` on it. And it was used only when doing audio uploads (with a background image).

I tried to investigate it now and it doesn't seem to have any effect by setting `const files = undefined`. But I can't understand how the backend is able to pair the image file with the audio file anymore. Before the only link between them was the audioBg filename sent by the client, but this doesn't seem to be the case anymore. @rigelk do you know if this has changed? Otherwise I'll investigate further tomorrow.

Reply:
Nevermind @kontrollanten I got it ; my initial refactorings had lost the meaning of your code regarding that part. My next commit will adress that.